pkg/validation/spec/responses.go

```diff
@@ -90,20 +90,28 @@ func (r ResponsesProps) MarshalJSON() ([]byte, error) {
 
 // UnmarshalJSON unmarshals responses from JSON
 func (r *ResponsesProps) UnmarshalJSON(data []byte) error {
-	var res map[string]Response
+	var res map[string]json.RawMessage
 	if err := json.Unmarshal(data, &res); err != nil {
-		return nil
+		return err
 	}
 	if v, ok := res["default"]; ok {
-		r.Default = &v
+		value := Response{}
+		json.Unmarshal(v, &value)
```

Comment:
don't we need to check for `err` here and return if this errors?

Reply:
whoops. thanks for catching that